apps/api/src/domain/generated/person/entities/person.entity.ts

```diff
@@ -36,6 +38,8 @@ export class Person {
   benefactors?: Benefactor[]
   beneficiaries?: Beneficiary[]
   campaignFiles?: CampaignFile[]
+  campaignReports?: CampaignReport[]
+  campaignReportFiles?: CampaignReportFile[]
```

Comment:
shouldn't the relation be from person to campaignReport and then from campaignReport to the files? 
In other words the Person should not have direct relation to the Files